api-tests/CMakeLists.txt

```diff
@@ -116,7 +116,7 @@ string(TOLOWER ${SUITE} SUITE_LOWER)
 # Check for valid targets
 _get_sub_dir_list(PSA_TARGET_LIST ${PSA_ROOT_DIR}/platform/targets)
 if(NOT ${TARGET} IN_LIST PSA_TARGET_LIST)
-	message(FATAL_ERROR "[PSA] : Error: Unspported value for -DTARGET=, supported targets are : ${PSA_TARGET_LIST}")
```

Comment:
could you please rebase and submit the changes again?

Reply:
Done